web_export_view/__manifest__.py

```diff
@@ -1,40 +1,27 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (C) 2012 Agile Business Group sagl (<http://www.agilebg.com>)
-#    Copyright (C) 2012 Domsense srl (<http://www.domsense.com>)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2016 Henry Zhou (http://www.maxodoo.com)
+# Copyright 2016 Rodney (http://clearcorp.cr/)
+# Copyright 2012 Agile Business Group
+# Copyright 2012 Domsense srl (<http://www.domsense.com>)
+# Copyright 2012 Therp BV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    'name': 'Export Current View',
-    'version': '8.0.1.2.0',
+    'name': "Web Export Current View",
+    'version': '10.0.1.0.0',
     'category': 'Web',
     'author': "Agile Business Group,Odoo Community Association (OCA)",
     'website': 'http://www.agilebg.com',
     'license': 'AGPL-3',
-    'depends': [
+    "depends": [
         'web',
     ],
-    'data': [
-        'view/web_export_view.xml',
+    "data": [
+        'views/web_export_view_view.xml',
     ],
     'qweb': [
-        'static/src/xml/web_export_view_template.xml',
+        "static/src/xml/web_export_view_template.xml",
     ],
-    'installable': False,
-    'auto_install': False,
+    'installable': True,
+    "auto_install": False,
```

Comment:
Why all these quote changes? Can't you leave them as they were and reduce the diff?